### DISCUSSION TITLE
RC receiver as BTlink to configure FC

### DISCUSSION THREAD

**Discussion by sandrosov85** [ExpressLRS/ExpressLRS#2594] [Ideas]
Hi guys!

Can RC receiver work without RC transmitter link as BTlink to configure BetaFlight or other software on FC? This can be wery usefull for tiny setup with hard access to USB port.

USECASE: (without RC Link from transmitter) connecting to wifi RC receiver -> click in interface button (somthing like "BT Link to FC") -> ESP setting up BT interface to FC that work until power up&down cycle. After reboot receiver work as usual...

**Comment by pkendall64** [comment]
The most common receiver is using the ESP8285 which does not have bluetooth. But, we do have configuration of BF over wifi.
Connect to the receiver, put it into wifi mode and BF configurator should detect the receiver in the Port selection box.
If your receiver is NOT configured to connect to the your home network you will have to connect your PC to the wifi access point that the receiver makes.
